docs/components/Address.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Address
+
+Display an address (or ENS) along with a utility icon to copy the address. If the address is associated with an ENS that has an avatar, this avatar will be displayed. If not, a blockie image representation of the address will be shown.
+
+By default, clicking on the address redirects you to the `targetNetwork` blockexplorer with the details of that address.
```

Comment:
Maybe we can add link `targetNetwork` to scaffold.config ? 

Similar to how we did for `pollingInterval` in `useScaffoldEventSubscriber` 

https://github.com/scaffold-eth/se2-docs/blob/85fd09ac9a74055e763c02ff2165fcc103b6f1df/docs/hooks/useScaffoldEventSubscriber.md?plain=1#L31

Reply:
Done! 🙌